### ISSUE TITLE
Invalid isoformat string when Month < 10 

### ISSUE THREAD

**Issue by stefancrain** [allangood/holiday_exporter#2]
**Error**
```
Traceback (most recent call last):
  File "/usr/bin/holiday_exporter.py", line 55, in <module>
    if date.fromisoformat(isodate) == date.today():
ValueError: Invalid isoformat string: '2020-1-01'
```

Using included yaml config (problem line)
```
...
  - date: "{YYYY}-{MM}-01"
    description: "Some event that repeats every month at specific day"
```

Isodate expects {MM} but `now.month` only gives us `1` causing the failure above. 
```
 isodate = custom_holiday['date'].upper().format(YYYY=now.year, MM=now.month)
```

**Comment by allangood** [comment]
Thank you for bring this issue!
I'm working in the fix and it will be available soon.

**Comment by allangood** [comment]
This bug is addressed on #3 
Thank you very much for your collaboration!
